maro/simulator/scenarios/data_center/common.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from .virtual_machine import VirtualMachine
+
+
+class Action:
+    """Data center scenario action object, which was used to pass action from agent to business engine.
+
+    Args:
+        vm (VirtualMachine): The virtual machine object, which only contains id, requirement resource, lifetime
+        pm_id (int): The physical machine id assigned to this vm.
+        buffertime (int): The buffer time to assign this VM.
+    """
+
+    def __init__(self, assign: bool, vm_req: VirtualMachine, buffer_time: int, pm_id: int = None):
+        self.assign = assign
+        self.vm_req = vm_req
+        self.pm_id = pm_id
+        self.buffer_time = buffer_time
+
+
+class Payload:
```

Comment:
if it is only for VM_REQUIREMENT, name it as sth. like VmRequirementPayload

Reply:
I think it might be the comment mislead you. At first, I implement Payload object for the requirement used only, but I thought it might can be re-used by finished payload or action payload. But it might be more clear separate them apart.